src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,23 @@
+import PropTypes from 'prop-types';
 import './MovieList.scss';
+import MovieCard from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard key={movie.imdbId} movie={movie} />)}
+  </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings